src/test/java/com/google/cloud/spanner/pgadapter/ProtocolTest.java

```diff
@@ -1387,72 +1379,60 @@ public void testCopyBatchSizeLimit() throws Exception {
     when(connectionHandler.getSpannerConnection()).thenReturn(connection);
     when(connection.execute(any(Statement.class))).thenReturn(statementResult);
     when(statementResult.getUpdateCount()).thenReturn(1L);
+    setupQueryInformationSchemaResults();
 
     byte[] payload = Files.readAllBytes(Paths.get("./src/test/resources/batch-size-test.txt"));
 
-    ResultSet spannerType = Mockito.mock(ResultSet.class);
-    Mockito.when(spannerType.getString("column_name")).thenReturn("key", "value");
-    Mockito.when(spannerType.getString("data_type")).thenReturn("bigint", "character varying");
-    Mockito.when(spannerType.next()).thenReturn(true, true, false);
-    Mockito.when(connection.executeQuery(any(Statement.class))).thenReturn(spannerType);
-
     CopyStatement copyStatement = new CopyStatement("COPY keyvalue FROM STDIN;", connection);
 
     Assert.assertFalse(copyStatement.isExecuted());
     copyStatement.execute();
-    Assert.assertTrue(copyStatement.isExecuted());
+    assertTrue(copyStatement.isExecuted());
```

Comment:
nit: This doesn't match the assertions below.  The class name is omitted here but not below and other references.

Assert.assertFalse
Assert.assertTrue

Reply:
Yeah, I know this looks a little messy at this moment. I tried to minimise the number of merge conflicts, so I didn't change the existing assertions where it was not necessary. My plan was to do a general overhaul of the assertions in the different test classes in a separate PR so they are all in the same style.